examples/templates/aws-devcontainer/README.md

```diff
@@ -0,0 +1,93 @@
+---
+display_name: AWS EC2 (Devcontainer)
+description: Provision AWS EC2 VMs with a devcontainer as Coder workspaces
+icon: ../../../site/static/icon/aws.png
+maintainer_github: coder
+verified: true
+tags: [vm, linux, aws, persistent, devcontainer]
+---
+
+# Remote Development on AWS EC2 VMs using a Devcontainer
+
+Provision AWS EC2 VMs as [Coder workspaces](https://coder.com/docs/v2/latest) with this example template.
+
+<!-- TODO: Add screenshot -->
```

Comment:
screenshot?

I left a comment on the GCP one it may be nice to show the architecture here https://docs.github.com/en/codespaces/overview